docs/source/gbd2017_models/causes/smoking_affected_causes/index.rst

```diff
@@ -0,0 +1,459 @@
+.. _2017_smoking_affected_causes:
+
+===============================================
+Smoking Affected Causes (Excluding Lung Cancer)
+===============================================
+
```

Comment:
since this is a mortality model, we can add mortality to the title of this documentation? "Modelling increased mortality associated smoking affected causes" or something like that...  ¯ \ _(ツ)_/¯